common/views/components/Body/Body.js

```diff
@@ -54,23 +54,18 @@ const Body = ({ body }: Props) => {
               </Layout8>
             }
 
-            {slice.type === 'picture' && slice.weight === 'default' &&
+            {slice.type === 'picture' &&
```

Comment:
Any chance of us putting this back?

Or does it interfere? I was hoping to put some examples together which this would negate.

Reply:
Should be fine, I think. Will do.